tools/create-image.sh

```diff
@@ -8,10 +8,67 @@ set -eux
 
 # Create a minimal Debian distribution in a directory.
 RELEASE=stretch
-DIR=stretch
+DIR=chroot
+FEATURE=minimal
+ADD_PACKAGE=""
+TRINITY=false
+PERF=false
+
+display_help() {
+    echo "Usage: $0 [option...] " >&2
+    echo
+    echo "   -d, --distribution         Set on which debian distribution to create"
+    echo "   -f, --feature              Check what packages to install in the image, options are minimal, full"
+    echo "   -p, --add-perf             Add perf support with this option enabled"
+    echo "   -t, --add-trinity          Add trinity support with this option enabled"
+    echo
+}
+
+while true; do
+    if [ $# -eq 0 ];then
+	echo $#
+	break
+    fi
+    case "$1" in
+        -h | --help)
+            display_help
+            exit 0
+            ;;
+        -d | --distribution)
+	    RELEASE=$2
+            shift 2
+            ;;
+        -f | --feature)
+	    FEATURE=$2
+            shift 2
+            ;;
+        -t | --add-trinity)
+            #echo "Add trinity Support"
+	    TRINITY=true
+            shift 1
+            ;;
+        -p | --add-perf)
+            #echo "Add perf Support"
+	    PERF=true
+            shift 1
+            ;;
+        -*)
+            echo "Error: Unknown option: $1" >&2
+            exit 1
+            ;;
+        *)  # No more options
+            break
+            ;;
+    esac
+done
+
+if [ $FEATURE = "full" ]; then
+    ADD_PACKAGE="make sysbench git vim tmux usbutils"
```

Comment:
I think it would be useful to allow user to also add own packages, because we can't foresee everything everybody wants to install. Thus it would be good to not override ADD_PACKAGE.